spec/order_spec.rb

```diff
@@ -0,0 +1,42 @@
+require 'order'
+
+RSpec.describe Order do
+
+  let(:menu) {double :menu, show_menu: {"chicken" =>3,
```

Comment:
Style/BlockDelimiters: Avoid using `{...}` for multi-line blocks. (https://github.com/bbatsov/ruby-style-guide#single-line-blocks)<br>Style/MultilineBlockLayout: Block body expression is on the same line as the block start.